src/hid_manager.rs

```diff
@@ -0,0 +1,42 @@
+use libc::c_void;
+use cf::{CFAllocatorRef, CFDictionaryRef, CFIndex};
+use types::IOOptionBits;
+use io_return::IOReturn;
+
+#[doc(hidden)]
+#[repr(C)]
+pub struct __IOHIDManager {
+    __private: c_void,
+}
+pub type IOHIDManagerRef = *const __IOHIDManager;
```

Comment:
This should probably be `*mut`. The C typedef doesn't specify `const`:

```c
typedef struct __IOHIDManager * IOHIDManagerRef;
```

And it's passed to functions that most likely mutate, e.g., `IOHIDManagerSetProperty()`.